### DISCUSSION TITLE
Return the number of queries

### DISCUSSION THREAD

**Discussion by Lengso** [projectdiscovery/uncover#9] [Ideas]
It is recommended to increase the number of query results returned by each query engine

For example I use all engines
```
shodan total: 2
fofa total: 10
censys total: 1
```

**Comment by ehsandeep** [comment]
@Lengso as default the result count set to 100 if available, if you wanted to increase the count, you update the number `l` flag or update the config file at `~/.config/uncover/config.yaml` to update any number you want `limit: 100`

**Reply by Lengso** [reply]
@ehsandeep You may have misunderstood, I don't mean to limit the number of returns, but to return the query statement corresponding to the total number of each engine, which is convenient for comparison

**Reply by ehsandeep** [reply]
Got it, yeah now I see what you mean, and yeah definitely useful information to display.
